packages/react-core/src/components/Compass/CompassHeader.tsx

```diff
@@ -0,0 +1,21 @@
+import styles from '@patternfly/react-styles/css/components/Compass/compass';
+import { css } from '@patternfly/react-styles';
+
+interface CompassHeaderProps {
+  /** Content of the logo area */
+  logo?: React.ReactNode;
+  /** Content of the navigation area */
+  nav?: React.ReactNode;
+  /** Content of the profile area */
+  profile?: React.ReactNode;
```

Comment:
Since these are all optional, we should conditionally render their elements below.

Reply:
wdyt @mcoker?

Should the logo/nav/profile wrapping divs be present regardless if content is pass to the respective prop?

Reply:
I think if we want to allow someone to pass `children` to `<CompassHeader>` and whatever they pass takes up the whole layout, yeah we would conditionally render them. I made logo/nav/profile optional in the core docs assuming you could do this. Though if they pass any one of the three, I think we need to render all 3 for the positioning/centering to work.